--- PKGBUILD	2019-11-30 00:24:59.467310217 +0100
+++ aspnet/PKGBUILD	2019-11-30 00:26:10.011653250 +0100
@@ -1,9 +1,8 @@
-pkgbase='dotnet3.0-bin'
-pkgname=('dotnet3.0-host-bin' 'dotnet3.0-runtime-bin' 'aspnet3.0-runtime-bin' 'dotnet3.0-sdk-bin')
+pkgname='aspnet3.0-runtime-bin'
 _runtimever=3.0.1
 _aspnetver=3.0.1
 _sdkver=3.0.101
-pkgver=${_runtimever/-/}
+pkgver=${_aspnetver/-/}
 pkgver=${pkgver//-/+}
 pkgrel=1
 arch=('x86_64')
